src/WebJobs.Script.Grpc/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -274,12 +274,6 @@ message RpcFunctionMetadata {
   // Raw binding info
   repeated string raw_bindings = 10;
 
-  // Retry Options: string representation of JObject retry options (maxRetryCount, intervals, etc.)
-  string retry_options = 11;
-
-  // Configuration Source: string representation of JToken configuration source property in function metadata
-  string config_source = 12;
-
   // unique function identifier (avoid name collisions, facilitate reload case)
   string function_id = 13;
```

Comment:
GRPC newbie question: Does the order change or properties has any impact? (11 and 12 are gone now)

Reply:
even 5 is not there in this message.

Reply:
Field numbers are an important part of Protobuf. They're used to identify fields in the binary encoded data, which means they can't change from version to version of your service. The advantage is that backward compatibility and forward compatibility are possible. Clients and services will ignore field numbers that they don't know about, as long as the possibility of missing values is handled.

https://docs.microsoft.com/en-us/dotnet/architecture/grpc-for-wcf-developers/protobuf-messages

Reply:
@soninaren Any thoughts on this?

Reply:
If I understood correctly, we shouldn't change the numbers to keep the protobuf backwards compatible

Reply:
Thank you!